src/amo/client.js

```diff
@@ -7,11 +10,18 @@ import App from './components/App';
 import sagas from './sagas';
 import createStore from './store';
 
-createClient(createStore, { sagas }).then(({ renderApp }) => {
+createClient(createStore, { sagas }).then(({ history, renderApp, store }) => {
   renderApp(App);
 
   if (module.hot) {
-    module.hot.accept('./components/App', () => {
+    module.hot.accept(['./sagas', './store'], () => {
+      // eslint-disable-next-line global-require
+      const { reducers } = require('./store');
+      const nextRootReducer = connectRouter(history)(combineReducers(reducers));
```

Comment:
To minimize bugs, I'd like to see `connectRouter(history)(combineReducers(reducers))` moved to a function like `createRootReducer()` (or something) so it can also be called [in the real store](https://github.com/mozilla/addons-frontend/blob/bc32a8c978894f7dc2fbd1135cdc710a07dd6aec/src/amo/store.js#L133).

Reply:
That totally makes sense :+1: